<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Black Scholes Merton European Pricer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>457</width>
      <height>322</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Fair</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Gamma (%)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="out_theta">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="out_vega">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="out_status">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Theta</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Dv01 / Rho</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="out_gamma">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Vega</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="out_fair">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Delta (%)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="out_rho">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="out_delta">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Spot</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="inp_spot">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Strike</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="inp_strike">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Maturity (Years)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="inp_maturity">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Volatility (%)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="inp_vol">
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Interest Rate (%)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="inp_intrate">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Dividend (%)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="inp_div">
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="inp_type">
           <item>
            <property name="text">
             <string>Call</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Put</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="QPushButton" name="btn_calc">
           <property name="text">
            <string>Calculate Fair and Greeks</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Inputs&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Outputs&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
